### PR TITLE
AA-230

### PR DESCRIPTION
**Pull request by MAzalya**
updated unit test Android Shimmer 2r

### CODE REVIEW

File: ShimmerAndroidInstrumentDriver/ShimmerAndroidInstrumentDriver/src/androidTest/java/com/shimmerresearch/android/AAPI_00001_Shimmer_GeneralBluetoothShimmer2R.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                    assertTrue(true);
+                } else {
+                    assertTrue(false);
+                }
+            }
+            //even value of i will stop streaming
+            else {
+                shimmer.stopStreaming();
+                Thread.sleep(DELAY_DURATION_MS);
+                if (shimmer.isStreaming()) {
```
Comment (nightscape) [inline ShimmerAndroidInstrumentDriver/ShimmerAndroidInstrumentDriver/src/androidTest/java/com/shimmerresearch/android/AAPI_00001_Shimmer_GeneralBluetoothShimmer2R.java]: The entire block can be simplified to `assertFalse(shimmer.isStreaming())`

File: ShimmerAndroidInstrumentDriver/ShimmerAndroidInstrumentDriver/src/androidTest/java/com/shimmerresearch/android/AAPI_00001_Shimmer_GeneralBluetoothShimmer2R.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+    @Test      //test for multiple start and stop streaming
+    public void test007_MultipleStartStopStreaming() throws Exception {
+        for (int i = 1; i < 8; i++) {
+
+            //odd value of i will start streaming
+            if (i % 2 != 0) {
+                shimmer.startStreaming();
+                Thread.sleep(DELAY_DURATION_MS);
+                if (shimmer.isStreaming()) {
```
Comment (nightscape) [inline ShimmerAndroidInstrumentDriver/ShimmerAndroidInstrumentDriver/src/androidTest/java/com/shimmerresearch/android/AAPI_00001_Shimmer_GeneralBluetoothShimmer2R.java]: The entire block can be simplified to `assertTrue(shimmer.isStreaming())`

Comment (JongChern) [reply ShimmerAndroidInstrumentDriver/ShimmerAndroidInstrumentDriver/src/androidTest/java/com/shimmerresearch/android/AAPI_00001_Shimmer_GeneralBluetoothShimmer2R.java:129]: good suggestions thanks

File: ShimmerAndroidInstrumentDriver/ShimmerAndroidInstrumentDriver/src/androidTest/java/com/shimmerresearch/android/AAPI_00001_Shimmer_GeneralBluetoothShimmer2R.java (hunk 1 above)
Comment (JongChern) [reply ShimmerAndroidInstrumentDriver/ShimmerAndroidInstrumentDriver/src/androidTest/java/com/shimmerresearch/android/AAPI_00001_Shimmer_GeneralBluetoothShimmer2R.java:139]: good suggestions thanks

**Comment by JongChern** [comment]
looks good accepting

**Review by JongChern — APPROVED** [review]

**Pull request merged**